src/modules/countlikes.js

```diff
@@ -0,0 +1,13 @@
+const countLike = (likesList, itemId) => {
```

Comment:
- [ ] [Optional] The like counter behaves strangely and doesn't register new likes when the user clicks the like button the number increment before sending the data to the API, it should get the data and display it on the page not increment it locally this gives false info to the user, also some times the counter gives ```undefined``` instead of the number of likes please consider fixing the like counter so it works as expected

- [ ] [Optional] For better UX please consider implementing a visual confirmation after the like is sent to the API, e.g: change the color of the like button (this is an optional suggestion)

![Peek](https://user-images.githubusercontent.com/25227022/169581343-d40da5fb-a28c-402d-abf4-57256e82b4f0.gif)